stdlib/3/multiprocessing/context.pyi

```diff
@@ -0,0 +1,5 @@
+# Stubs for multiprocessing.context
+
+# NOTE: These are incomplete!
+
+class BaseContext(object): ...
```

Comment:
Incomplete stubs aren't well supported; with this stub as written mypy won't let you call any method on a BaseContext, for example. Therefore, I don't think we should add any new incomplete stubs. We should either complete this stub or revert the `ctx` argument to `Any` with a note to change it to BaseContext once there is a stub for `multiprocessing.context`.

Sorry for the back and forth and for the delay; I was out for a few days.